bftengine/src/bftengine/ReplicaImp.cpp

```diff
@@ -2777,14 +2796,14 @@ void ReplicaImp::onInfoRequestTimer(Timers::Handle timer) {
 }
 
 void ReplicaImp::onSuperStableCheckpointTimer(concordUtil::Timers::Handle) {
-  if (!isSeqNumToStopAt(lastExecutedSeqNum)) return;
-  if (!checkpointsLog->insideActiveWindow(lastExecutedSeqNum)) return;
-  CheckpointMsg *cpMsg = checkpointsLog->get(lastExecutedSeqNum).selfCheckpointMsg();
+  if (!isSeqNumToStopAt(lastStableSeqNum)) return;
```

Comment:
 This wasn't really a bug, because if we stopped on lastExecutedSeqNum then eventually it will become lastStableSeqNum. 
It is just more comprehensive with lastStableCheckpoint